### ISSUE TITLE
Create Local Boolean Latch logix node

### ISSUE THREAD

**Issue by mralext20** [Frooxius/NeosPublic#908]
this can already be achived with a boolean latch and a user Value Override pointed at the state, but having this as a first class node would be pretty cool as well.

some example use cases is having checks for individual users doing things, like completing a small math problem or pressing keys.

**Comment by H3BO3** [comment]
This would likely fall into the category of local variables, and froox has said they are a prerequisite for collections, he just isn't prioritizing it at the moment.

**Comment by Frooxius** [comment]
Yeah this definitely needs to wait for the local variable/value system first to be handled properly, otherwise it can lead to messy patterns and confusion.
